src/extensionsIntegrated/CSSColorPreview/main.js

```diff
@@ -69,22 +69,26 @@ define(function (require, exports, module) {
         for (let i = 0; i < nLen; i++) {
             let lineText = editor.getLine(i);
 
-            if ((lineText.indexOf('/*') !== -1) || (lineText.indexOf('*/') !== -1)) {
+            // to skip commented lines
+            // The second parameter in the getToken() method, `true`, enables precise tokenization.
+            // Without setting `precise` to `true`, multi-line comments are not handled correctly.
+            const token = editor.getToken({line: i}, true);
```

Comment:
will this break inside inline comments? For eg, in line `border none; /*color: red; */` , the first token is not a comment and will pass through mostly.
The check should likeley be done at the start pos of individual colors array below.verify that usecase.